config/config.yaml

```diff
@@ -15,6 +15,8 @@ cerberus:
         -    openshift-kube-scheduler
         -    openshift-ingress
         -    openshift-sdn                               # When enabled, it will check for the cluster sdn and monitor that namespace
+    watch_schedulable_masters_labels:                    # List of master node labels to watch, add more labels as required with True/False boolean to enable/disable.
```

Comment:
Should we rename this option to be more generic? Still will want to check that the nodes are set to NoSchedule.
Maybe just _watch_schedulable_labels_? Definitely should call out that we are looking for a the nodes with our given label should be not schedulable

Reply:
@paigerube14 updated the var name and structure.